src/Raven.Client/Constants.cs

```diff
@@ -220,6 +220,8 @@ private Querying()
                 }
 
                 public const string IntersectSeparator = " INTERSECT ";
+
+                public const string DynamicQueryPrefix = "dynamic/";
```

Comment:
Hm... I'm not sure that this is a good idea.
Might be better to do:

`Query<T>(string indexName = null, string collection = null);`

Then you can query it like:

`session.Query<dynamic>(collection: "Users")`

Vs.

`session.Query<T>("Users/ByName");`

Thoughts?